GrandCentralBoard/Widgets/Slack/View/MessageBubbleView.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  MessageBubbleView.swift
+//  GrandCentralBoard
+//
+//  Created by Michał Laskowski on 25.05.2016.
+//  Copyright © 2016 Macoscope. All rights reserved.
+//
+
+import UIKit
+
+
+@IBDesignable
+final class MessageBubbleView: UIView {
+
+    private lazy var imageView: UIImageView = { [unowned self] in
+        let imageView = UIImageView(image: UIImage(named: "message_bubble")!)
+        self.addSubview(imageView)
+        return imageView
+    }()
+
+    private lazy var label: UILabel = { [unowned self] in
+        let label = UILabel()
+        label.font = UIFont.systemFontOfSize(40)
+        label.textColor = UIColor.whiteColor()
+        label.numberOfLines = 0
+
+        self.addSubview(label)
+        return label
+    }()
+
+    @IBInspectable var text: String = "" {
+        didSet {
+            label.text = text
+            setNeedsLayout()
+        }
+    }
+
+    override func layoutSubviews() {
```

Comment:
I'm like super busy right now but this doesn't sound right and this code is archaic. 

(did you unchecked the margin checkbox?)